lldb/source/Plugins/ExpressionParser/Clang/CxxModuleHandler.cpp

```diff
@@ -280,7 +280,8 @@ std::optional<Decl *> CxxModuleHandler::tryInstantiateStdTemplate(Decl *d) {
       new_class_template->getDeclContext(),
       new_class_template->getTemplatedDecl()->getLocation(),
       new_class_template->getLocation(), new_class_template, imported_args,
-      nullptr);
+      td->hasMatchedPackOnParmToNonPackOnArg(),
+      /*PrevDecl=*/nullptr);
```

Comment:
@DavidSpickett @Michael137 this is the only change for this relanding of the patch.

As far as I understood reading this code, this is basically importing a template instantiation into a different context, so this is much like what we would do at the AST Importer, ie just forward the new bool from the original specialization into this new one.

Also as I understand, this implementation whitelists only a few template names from the STL, none of which have template template parameter to begin with, so this new flag will always be false here and this is pretty much untestable.

Reply:
yup this is correct, thanks